ci/si_pipeline.sh

```diff
@@ -29,7 +29,7 @@ function create-junit-xml {
 }
 
 function exit-as-unstable {
-    echo "Cluster launch failed."
+    echo "$1"
     create-junit-xml "dcos-launch" "cluster.create" "$1"
     ./dcos-launch delete
     exit 0
```

Comment:
why do we exit with 0 here in case of error?

Reply:
Ah, because a launch failure should be an unstable build not a broken build.

Reply:
ok, makes sense! :) thanks